FirebaseVertexAI/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# 0.2.0
```

Comment:
I think _this_ should be `10.26.0`. @paulb777 do you know for sure?

Reply:
Yes. Good catch.

Reply:
Done.